Project.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+using namespace std;
+
+class User{
+    public:
+    string Username;
+    string Email;
+    User(string username,string email){
+        Username = username;
+        Email = email;
+    }
+    void print(){
+        std::cout<<"Username - "<<Username<<endl;
+        std::cout<<"Email - "<<Email<<endl;
+    }
+    void changeUsername(string newUsername){
+        Username = newUsername;
+    }
+};
+
+class Note{
+    public:
+    string TopicName;
+    string NoteContent;
+    Note(string topic, string noteContent){ 
+        TopicName = topic;
+        NoteContent = noteContent; 
+    }
+    void print(){
+        std::cout<<"Topic - "<<TopicName<<endl;
+        std::cout<<"NoteContent - "<<NoteContent<<endl;
+    }
+    void changeTopic(string newTopic){
+        TopicName = newTopic;
+    }
+};
+int main(){
```

Comment:
Implement exception handling to manage runtime errors, especially when dealing with user inputs or file operations. [important]